### DISCUSSION TITLE
使用本地运行的LLM作为翻译服务

### DISCUSSION THREAD

**Discussion by pto2k** [immersive-translate/immersive-translate#1394] [Ideas]
### 新功能 | New Feature

希望能使用本地运行的LLM作为翻译服务

### 为什么您想要添加这些功能 | Why you want to add these features

为了某些网络安全要求，需要翻译的文本（例如会议的字幕）不能外传到DeepL或OpenAI等平台。所以本地运行LLM是一个解决方案。

### 提议 | Proposal

接入一些比较有人气的并且提供API的Local LLM App。
例如LMStudio
https://medium.com/@ingridwickstevens/running-a-local-openai-compatible-mixtral-server-with-lm-studio-cd602efbf808

**Comment by theowenyoung** [comment]
您可以利用自定义 API 来接入任何翻译服务： https://immersivetranslate.com/docs/services/custom/
